pytest_translations.py

```diff
@@ -147,6 +158,141 @@ def reportinfo(self):
         return (self.fspath, -1, "po-fuzzy")
 
 
+if enchant:
+    lang_dicts = [name for name, _ in enchant.list_dicts()]
+else:
+    lang_dicts = []
+
+
+class StringFormatChunker(Chunker):
+    """Chunker for breaking up HTML documents into chunks of checkable text.
+
+    The operation of this chunker is very simple - anything between a "<"
+    and a ">" will be ignored.  Later versions may improve the algorithm
+    slightly.
+    """
+
+    def next(self):
+        text = self._text
+        offset = self.offset
+        while True:
+            if offset >= len(text):
+                break
+            #  Skip to the end of the current tag, if any.
+            if text[offset] == "{":
+                maybeTag = offset
+                if self._is_tag(text,offset):
+                    while text[offset] != "}":
+                        offset += 1
+                        if offset == len(text):
+                            offset = maybeTag+1
+                            break
+                    else:
+                        offset += 1
+                else:
+                    offset = maybeTag+1
+            sPos = offset
+            #  Find the start of the next tag.
+            while offset < len(text) and text[offset] != "{":
+                offset += 1
+            ePos = offset
+            self._offset = offset
+            # Return if chunk isnt empty
+            if(sPos < offset):
+                return (text[sPos:offset],sPos)
+        raise StopIteration()
+
+    def _is_tag(self,text,offset):
+        if offset+1 < len(text):
+            if text[offset+1].isalpha():
+                return True
+        return False
+
+
+class OldStringFormatChunker(Chunker):
```

Comment:
tests missing

and it's nearly a 100% duplication of the `StringFormatChunker`